src/components/Breadcrumbs/index.tsx

```diff
@@ -4,29 +4,37 @@ import { findDirectoryNode as findNode } from '@/utils/findDirectoryNode';
 import classNames from 'classnames';
 
 type BreadcrumbItem = {
-  href: { pathname: string, query?: { platform: string } };
+  href: { pathname: string; query?: { platform: string } };
```

Comment:
I think my prettier updated this to a semicolon because this is a "type" which should use semicolons